source/styleguide/molecules/SgPageShell/SgPageShell.jsx

```diff
@@ -22,13 +20,7 @@ export const SgPageShell__navigation = (props) => {
 
   return (
     <nav className={classStack} {...attrs}>
-      <Scrollbars
```

Comment:
Lets leave the custom scroll bars please. This is for Windows users like me who don't get nice fancy thin OSX scroll bars :D